src/read/macho/load_command.rs

```diff
@@ -381,6 +381,32 @@ impl<E: Endian> macho::SymtabCommand<E> {
     }
 }
 
+impl<E: Endian> macho::LinkeditDataCommand<E> {
```

Comment:
I wonder if using `LinkeditDataCommand` for all of these was a mistake. Is there any common functionality that they share other than the load command header format? Not something to change in this PR though.